test/scripts/setup_test_environment.sh

```diff
@@ -8,5 +8,6 @@ if [[ ${CI} == "true" && (${NODE_ENV} = "test" && ${TRAVIS_PULL_REQUEST} != "fal
   sudo ./test/scripts/install_chromedriver_travis.sh
 
   chmod 600 ~/.ssh/monster_rsa
-  gem install bundler
+  sudo apt-get install ruby-full
```

Comment:
Why install ruby here?

Reply:
Because we only need to run bundler for the test environment, if we do it for every environment where it's not needed, it will slow down the build completion time.

Reply:
But if the gem install bundler was working before then it means that a ruby version is already installed. You should be fine with just the change to the gem install bundler line.

Reply:
Yes, that's true, it was using the pre-installed versions of ruby on the travis environment. My idea here was that it might be worth using a specific version so that we always know which ruby version we are relaying on. This is using the latest stable ruby version for linux which is 2.3.1 (bundler v2.0.0 was still failing with this one btw). I'm not sure how much value we are getting by adding this line though, it will also slow down the build.

Reply:
Installing this Ruby version doesn't make it the Ruby version that is currently linked so you'd still be using version 2.x which is why this isn't working. I believe Travis uses `rvm` so you'd need to switch to the new Ruby version with `rvm use x.x.x`.

Reply:
However, I'd just get rid of this line as your change on line 12 should fix the issue.

Reply:
Yes, installing ruby in the `before_install` of `travis.yml` is pretty straight forward and it uses RVM. I was able to install ruby 2.6.0 and ruby 2.4.0 but bundler v2.0.0 was still returning an error. Anyways, using bundler < 2 works so I will get rid of the ruby installation for now and handle that separately.